src/libraries/System.Linq/src/System/Linq/Range.cs

```diff
@@ -73,27 +101,33 @@ public override void Dispose()
             {
                 _state = -1; // Don't reset current
             }
+
+            internal static int StartMaxCount(T start, T max)
```

Comment:
I'm guessing this is effectively trying to measure a distance between `start` and `max`? I'm not sure if this is necessarily a well-defined operation for every kind of binary integer type (e.g. Gaussian integers)

cc @tannergooding